src/Proxy.NET/Routes/Attendee.cs

```diff
@@ -0,0 +1,55 @@
+using Microsoft.AspNetCore.Mvc;
+using Proxy.NET.Models;
+using Proxy.NET.Services;
+
+namespace Proxy.NET.Routes;
+
+public static class Attendee
+{
+    public static RouteGroupBuilder MapAttendeeRoutes(this RouteGroupBuilder builder)
+    {
+        var attendeeGroup = builder.MapGroup("/attendee/event/{eventId}");
+        attendeeGroup.MapPost("/register", AttendeeAdd).WithMetadata(new Auth(Auth.Type.Jwt));
+        attendeeGroup.MapGet("/", AttendeeGetKey).WithMetadata(new Auth(Auth.Type.Jwt));
+        return builder;
+    }
+
+    /// <summary>
+    /// Adds an attendee to the specified event.
+    /// </summary>
+    /// <param name="attendeeService">The attendee service.</param>
+    /// <param name="context">The HTTP context.</param>
+    /// <param name="eventId">The ID of the event.</param>
+    /// <returns>The task result contains the API key of the added attendee.</returns>
+    private static async Task<IResult> AttendeeAdd(
+        [FromServices] IAttendeeService attendeeService,
+        [FromServices] IRequestService requestService,
+        HttpContext context,
+        string eventId
+    )
+    {
+        string? userId = requestService.GetRequestContext() as string;
```

Comment:
Is there a reason why this is `string?` if we're just forcing it on use as `userId!`?

If we don't want null, it shouldn't return a nullable here.

Reply:
I was having a null issue in requestService and I also just read that as string is a safe cast with fall back to null, so I now just cast as string.. and removed the null overrides